app/components/SearchHistoryItem/styles.js

```diff
@@ -10,4 +10,11 @@ export const SearchHistoryItemWrapper = styled.li`
   &:not(:last-child):after {
     content: ',';
   }
+
+  a.active {
```

Comment:
Wouldn't `a:active` work?

Reply:
Nope, the active class is added by NavLink react-router-dom when the link/page is active. There's not active attribute on the element.